image.go

```diff
@@ -528,6 +533,8 @@ func (b *Builder) makeImageRef(manifestType string, exporting bool, compress arc
 		annotations:           b.Annotations(),
 		preferredManifestType: manifestType,
 		exporting:             exporting,
+		onBuild:               b.GetOnBuild(),
```

Comment:
This shouldn't be necessary - the value should be written to the image along with the rest of the image configuration.